docs/config/aws_template.md

```diff
@@ -0,0 +1,440 @@
+# Deploying to AWS
+
+!!! danger "AWS Admin Privileges Required"
+
+    This deployment can only be run by AWS administrators, so we assume the user has a basic understanding of code
+    development, Docker containers, and the AWS cloud.
+
+## Overview
+
+The goal of this guide is to help you create a functioning pipeline in AWS that allows you to process data on-the-fly as
```

Comment:
`The goal of this guide is to help you deploy your Tsdat pipeline(s) to AWS so that you can process data on-the-fly [keep rest of sentence]`

Reply:
Yep, that sounds better too. Made this change